Library/Homebrew/livecheck/strategy/crate.rb

```diff
@@ -0,0 +1,115 @@
+# typed: true
+# frozen_string_literal: true
+
+module Homebrew
+  module Livecheck
+    module Strategy
+      # The {Crate} strategy identifies versions of a crates.io package by
+      # checking the information from the `versions` API endpoint.
+      #
+      # Crate URLs have the following format:
+      #   `https://static.crates.io/crates/example/example-1.2.3.crate`
+      #
+      # The default regex identifies versions like `1.2.3`/`v1.2.3` from the
+      # version `num` field. This is a common version format but a different
+      # regex can be provided in a `livecheck` block to override the default
+      # if a package uses a different format (e.g. `1.2.3d`, `1.2.3-4`, etc.).
+      #
+      # @api public
+      class Crate
+        # The default regex used to identify versions when a regex isn't
+        # provided.
+        DEFAULT_REGEX = /^v?(\d+(?:\.\d+)+)$/i
+
+        # The default `strategy` block used to extract version information when
+        # a `strategy` block isn't provided.
+        DEFAULT_BLOCK = proc do |json, regex|
+          json["versions"]&.map do |version|
+            next if version["yanked"] == true
+
+            match = version["num"]&.match(regex)
+            next if match.blank?
```

Comment:
```suggestion
            next unless match
```
Nit, but more idiomatic (and less reliant on monkey patches). (I think this applies to every use of `blank?` in this PR.)

Reply:
Thanks for the feedback. I'll incorporate this on the next push.

We use the `next if match.blank?` pattern in a bunch of `livecheck` blocks, so I'll have to look into updating existing `strategy` blocks in a similar fashion.